Sources/OpenAPIServiceModel/CreateOpenAPIServiceModel.swift

```diff
@@ -0,0 +1,382 @@
+                    bodyStructureName = refName
+                }
+            }
+        }
+    }
+    
+    static func ignoreRequestHeader(operationName: String, headerName: String,
+                                     modelOverride: ModelOverride?) -> Bool {
+        
+        guard let ignoreRequestHeaders = modelOverride?.ignoreRequestHeaders else {
+            return false
+        }
+        
+        if ignoreRequestHeaders.contains("*.*") {
+            return true
+        }
+        
+        if ignoreRequestHeaders.contains("*.\(headerName)") {
+            return true
+        }
+        
+        if ignoreRequestHeaders.contains("\(operationName).\(headerName)") {
+            return true
+        }
+        
+        if ignoreRequestHeaders.contains("\(operationName).*") {
+            return true
+        }
+        
+        return false
+    }
+    
+    static func getFixedFieldsOperationMembers(fixedFields: OpenAPI.Parameter, operationName: String,
+                                               index: Int, members: inout OpenAPIServiceModel.OperationInputMembers,
+                                               items: JSONSchema, model: inout OpenAPIServiceModel, modelOverride: ModelOverride?) {
+        let typeName = fixedFields.name.safeModelName().startingWithUppercase
+        
+        let fieldName = "\(operationName)Request\(typeName)"
+        let member = Member(value: fieldName,
+                            position: index,
+                            required: fixedFields.required,
+                            documentation: fixedFields.description)
+        switch fixedFields.location {
+        case .query:
+            members.queryMembers[fixedFields.name] = member
+        case .path:
+            members.pathMembers[fixedFields.name] = member
+        case .header:
+            guard !ignoreRequestHeader(operationName: operationName, headerName: fixedFields.name, modelOverride: modelOverride) else {
+                return
+            }
+            
+            members.additionalHeaderMembers[fixedFields.name] = member
+        default:
+            break
+        }
+        
+        addField(item: items, fieldName: fieldName,
+                 model: &model, modelOverride: modelOverride)
+    }
+    
+    static func addOperationResponseFromSchema(schema: JSONSchema, operationName: String, forCode code: Int, index: Int?,
+                                               description: inout OperationDescription,
+                                               model: inout OpenAPIServiceModel, modelOverride: ModelOverride?, document: OpenAPI.Document) {
+        switch schema {
+        case .object:
+            let indexString = index?.description ?? ""
+            var structureName = "\(operationName)\(code)Response\(indexString)Body"
+            var structureDescription = StructureDescription()
+            guard let objectContext = schema.objectContext else {
```

Comment:
It seems like here we throw an error if the object context is missing but in line 171 (same file) we do not.  Is this the correct behavior?

Reply:
Since we are outside of a loop I decided to put that guard since we can't use continue. That being said, the alternative to throwing would just be encapsulating the below lines in an if let if you think that would be better.

Reply:
Got it, I guess what I'm after is should this be an error case (in which case we should throw) or should we encapsulate it in an "if let" (and not throw)?

Reply:
I think for consistency I will put this in an if let